deployment/keystone/changeset/test/env_setup.go

```diff
@@ -127,6 +120,49 @@ func (te EnvWrapper) CapabilityInfos() []kcr.CapabilitiesRegistryCapabilityInfo
 	return caps
 }
 
+func (te EnvWrapper) OwnedCapabilityRegistry() *changeset.OwnedContract[*kcr.CapabilitiesRegistry] {
+	return loadOneContract[*kcr.CapabilitiesRegistry](te.t, te.Env, te.Env.Chains[te.RegistrySelector], registryQualifier)
+}
+
+func loadOneContract[T changeset.Ownable](t *testing.T, env deployment.Environment, chain deployment.Chain, qualifier string) *changeset.OwnedContract[T] {
+	t.Helper()
+	addrs := env.DataStore.Addresses().Filter(datastore.AddressRefByQualifier(qualifier))
+	require.Len(t, addrs, 1)
+	c, err := changeset.GetOwnedContractV2[T](env.DataStore.Addresses(), chain, addrs[0].Address)
+	require.NoError(t, err)
+	require.NotNil(t, c)
+	return c
+}
+
+func (te EnvWrapper) CapabilityRegistryAddressRef() datastore.AddressRefKey {
+	addrs := te.Env.DataStore.Addresses().Filter(datastore.AddressRefByQualifier(registryQualifier))
+	require.Len(te.t, addrs, 1)
+	return datastore.NewAddressRefKey(addrs[0].ChainSelector, addrs[0].Type, addrs[0].Version, addrs[0].Qualifier)
```

Comment:
You should be able to use the `Key()` method on the `AddressRef` [implementation](https://github.com/smartcontractkit/chainlink-deployments-framework/blob/5ee7a55e5bba098536e0ed751075c72bad9347c1/datastore/address_ref.go#L58) instead of constructing the key manually.
```suggestion
	return addrs[0].Key()
```